web/packages/test/scripts/configure-base-fee.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -eu
+
+source scripts/set-env.sh
+
+configure_base_fee() {
+    pushd $root_dir/smoketest
+    ./make-bindings.sh
+    cargo test --test configure_base_fee
```

Comment:
Just curious about the decision to use the smoke test here, instead of using polkadot-js like in `xcm-helper.sh`?

Reply:
I would assume the shell script only applies to some simple scenario, for configuring complex data structures or some business logic required script based on a high-level language like Javascript or Rust is better. As we've already heavily used rust for the smoke test I would prefer the latter.

Reply:
In principle I agree, but I do worry we are creating dependencies all over the place that are not that obvious. For example, same as my comment here: https://github.com/Snowfork/snowbridge/pull/940#discussion_r1338677303. Now the smoketests depend on some contract scripts and our bash scripts depend on the smoketests. These dependencies aren't always clear and it would be nice if each part of our project is mostly self-contained.

Reply:
https://github.com/Snowfork/snowbridge/pull/940#discussion_r1339379792

Reply:
That's okay with me, if we can reduce the inter-dependencies between the E2E script and the smoke tests in a follow-up PR.